recipes-extended/ostree/ostree_%.bbappend

```diff
@@ -0,0 +1,10 @@
+PACKAGECONFIG_append = " curl libarchive"
+PACKAGECONFIG_class-native_append = " curl"
```

Comment:
Should we remove `soup` from PACKAGECONFIG (on by default), as we use curl instead?

Reply:
I had that initially but removed as curl takes priority and doesn't really affect the build, besides having another dependency. I can remove to make it the same as before, will give it another shot locally.